res/battle/graphic/pl_batt_obj/meson.build

```diff
@@ -0,0 +1,23 @@
+subdir('prebuilt')
+subdir('palettes')
+subdir('sprites')
+
+pl_batt_order = files('order.txt')
+
+nitrofs_files += custom_target('pl_batt_obj.narc',
+    output: 'pl_batt_obj.narc',
+    input: [
+        pl_batt_order,
+        pl_batt_obj_nclr,
+        pl_batt_obj_built,
+        pl_batt_obj_ncgr
+    ],
+    command: [
+        knarc_exe,
+        '-d', '@PRIVATE_DIR@',
+        '-o', '@INPUT0@',
+        '-p',
+        '@OUTPUT@'
+    ],
+    build_always_stale: true
```

Comment:
`build_always_stale` should not be required here, but I had some issues while setting this up where changes to the `data.json` files (which stores settings for each item being built) wasn't triggering a rebuild of the NARC at all. maybe a Meson wizard can fix this up!